packages/nimbus/src/components/nimbus-provider/locale-context.tsx

```diff
@@ -0,0 +1,5 @@
+import { createContext, useContext } from "react";
+
+export const LocaleContext = createContext<string>("en-US");
```

Comment:
We could also use this to provide things such as i18n providers/formatters, etc

Reply:
also, `@internationalized/string`'s `useLocalizedStringFormatter` is [really interesting](https://github.com/adobe/react-spectrum/pull/3294).  We could consider using it internally since `react-aria` is already.

Alternately, since we use `react-intl` in the MC, we could also specify it as a peer-dep if we want to use good ol `intl.formatMessage`'n'frenz

Reply:
@tylermorrisford we sure can